README.md

```diff
@@ -7,7 +7,7 @@
 - [OpenID Connect Community Docker image on dockerhub](https://hub.docker.com/r/unleashorg/unleash-community-openid-connect)
 - [Official Unleash Helm Chart on artifacthub](https://artifacthub.io/packages/helm/unleash/unleash)
 
-**Steps:**
+**Steps: **Azure** **
```

Comment:
I would keep this intact (this also breaks MD) and instead change the paragraph in line 23 to:

"See `index.js` and `auth-hook.js` example implementations for other auth providers within the [example repository](https://github.com/Unleash/unleash-examples/tree/main/v4/securing-azure-auth). E.g., for Azure, it would look like this:"

```suggestion
**Steps:**
```